custom_components/tuya_local/devices/soleusair_air_purifier.yaml

```diff
@@ -0,0 +1,127 @@
+name: Soleusair Air Purifier
+products:
+  - id: kik7qavg2baebtk9
+    name: Soleusair Air Purifier KJ1000F A10
+  - id: 5xrkvysk7z6xrjum
+    name: Soleusair Air Purifier KJ760F A02
+
+primary_entity:
+  entity: switch
+  name: power
+  icon: "mdi:power"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+
+secondary_entities:
+  - entity: select
+    name: mode
+    category: config
+    dps:
+      - id: 3
+        type: string
+        name: option
+        mapping:
+          - dps_val: manual
+            value: manual
+          - dps_val: auto
+            value: auto
+          - dps_val: sleep
+            value: sleep
+          - dps_val: displayoff
+            value: displayoff
+  - entity: number
+    name: windspeed
+    category: config
+    dps:
+      - id: 101
+        type: integer
+        name: value
+        range:
+          min: 1
+          max: 5
+        unit: gear
+  - entity: sensor
+    name: filter_life
```

Comment:
Names should be intended for UI display: "Filter life" with spaces and capitalisation of the first word per HA naming guidelines.

https://developers.home-assistant.io/blog/2022/07/10/entity_naming/